src/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to="/" />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route index />
+          <Route path=":slug" />
+        </Route>
```

Comment:
The `PeoplePage` route includes a nested route with a `path=":slug"`, but it does not specify an element to render. If this is intentional, consider adding a comment for clarity. Otherwise, ensure that an appropriate component is provided for this route.